app/views/notes/index.html.erb

```diff
@@ -39,13 +39,5 @@
   <%= render :partial => "notes/notes" %>
 
   <hr />
-  
 </div>
 
```

Comment:
The sidebar is also used in the admin/queue section, which reuses the 'notes/index' template, not sure if it should be removed.

Reply:
It also removes the write research note button
![Screenshot 2020-12-09 at 22 32 57](https://user-images.githubusercontent.com/7622875/101678068-7f362300-3a6e-11eb-832b-dfc8765e7fe8.png)